.circleci/config.yml

```diff
@@ -49,8 +49,8 @@ jobs:
     <<: *test
 
 workflows:
-  version: 2
-  my_app:
+  version: 1
```

Comment:
why downgrading the version? this might have side effects:
https://circleci.com/docs/2.0/configuration-reference/#version-1